x-pack/plugin/inference/build.gradle

```diff
@@ -27,6 +27,12 @@ base {
   archivesName = 'x-pack-inference'
 }
 
+versions << [
+  'awsbedrockruntime': '1.12.740',
+  'aws'              : '1.12.270',
+  'jackson'          : '2.15.0'
```

Comment:
Probably not worth spending too much time on this but just curious if we're able to remove some references to jackson? We can totally do this after this PR is merged too.

Reply:
I just tried a test, and I _think_ we can... at least, it appears to work without the `aws` and `jackson` entries here...